test/unit/kobjectstore_test.rb

```diff
@@ -2054,6 +2055,47 @@ def test_time_constraints_and_result_limits
     assert_equal 31, q_over_limit.execute(:ref, :any).length
   end
 
+  def test_offsetting
+    restore_store_snapshot("min")
+    # Make some test objects with sorting dates set at day spacing.
+    time_base = Time.utc(2006, 06, 01, 12, 00)
+    0.upto(30) do |day|
+      o = KObject.new()
```

Comment:
Nit: the o and 0 is throwing me off visually, recommend giving the kObject a proper name instead of o?

Reply:
Copy from Haplo tests above to keep format consistent.